src/porepy/viz/exporter.py

```diff
@@ -180,6 +182,183 @@ def __init__(
         # Parameter to be used in several occasions for adding time stamps.
         self._padding = 6
 
+    def import_from_vtu(
```

Comment:
So the idea is that if I have a `MixedDimensionalGrid`, I can create an `Exporter` and use this function to populare the `mdg` state? I like the idea, and although we can certainly discuss whether this is the right placement of such functionality, I can live with a two-stage approach.

Reply:
You are addressing two issues:
1. The two stage process: Yes, indeed, the `MixedDimensionalGrid` is assumed for now, i.e., no mesh creation from vtu or projection of data from non-matching grids is covered in this addition. For now, I assumed the simplest applications possible.
2. The placement of the importer functionality. Indeed, when starting developing, I started with a separate file. However, in order to double check whether a vtu file is complying with the current `MixedDimensionalGrid`, many of the already predetermined quantities of the `Exporter`  are required. Thereby, as you also commented somewhere else, the `Exporter` becomes more an two-way interface bewteen meshio and porepy.

Reply:
Sure. (Note that I am not familiar with pickle). Can't the gmsh file be used to recreate a `MixedDimensionalGrid`. I only used this functionality for `MixedDimensionalGrids` with constraints and not true mortar grids. Or is there a difference between using pickle and reading from gmsh?

Reply:
Pickle may not be optimal, since it is very particular that the object is loaded into the exact same environment it was made. So if you pickle an `mdg`, pull an update from GH, and then load the pickled file, you risk compatibility errors (which is probably fair from pickle's side, even though we know the grid is the same). Gmsh should be safe, but then you need to do all the splitting again, and this takes time for large domains. I have some other ideas, but this is not the PR to do that.

I suggest we keep this for now, and then revisit import/export later on.

I'm fine with resolving this comment unless @IvarStefansson has other opinions.

Reply:
The updated code does not address the discussed issues mentioned here. It is still further assumed, that the `MixedDimensionalGrid` is provided and one wishes to attach data to the grid read from a complying vtu file, to be checked for compatibility.

Reply:
Looking at this now, it is an interesting discussion, but off-topic for this PR. We should come back to the question of how to do grid import soon, but not here.